composer.json

```diff
@@ -14,7 +14,25 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "sonata-project/admin-bundle": "^3.1",
+        "sonata-project/media-bundle": "^3.0",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.0",
+        "symfony/class-loader": "^2.3 || ^3.0",
```

Comment:
It's a new bundle, so no reason to support the old versions. We will drop symfony <2.8 with the next major releases on all bundles.